go/worker/compute/executor/init.go

```diff
@@ -1,16 +1,42 @@
 package executor
 
 import (
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
 	workerCommon "github.com/oasisprotocol/oasis-core/go/worker/common"
 	"github.com/oasisprotocol/oasis-core/go/worker/compute"
+	txnSchedulerAlgorithm "github.com/oasisprotocol/oasis-core/go/worker/compute/executor/committee/txnscheduling/algorithm"
 	"github.com/oasisprotocol/oasis-core/go/worker/registration"
 )
 
+const (
+	// CfgScheduleCheckTxEnabled enables checking each transaction before
+	// scheduling it.
+	CfgScheduleCheckTxEnabled = "worker.exectutor.schedule_check_tx.enabled"
+)
+
+// Flags has the configuration flags.
+var Flags = flag.NewFlagSet("", flag.ContinueOnError)
+
+// ScheduleCheckTxEnabled reads CfgScheduleCheckTxEnabled flag from viper.
+func ScheduleCheckTxEnabled() bool {
```

Comment:
Is there a reason why this needs to be public?

Reply:
hm it was public before, but yeah doesn't seem like it needs to be.